src/types/comfyWorkflow.ts

```diff
@@ -1,6 +1,15 @@
 import { z } from 'zod'
 import { fromZodError } from 'zod-validation-error'
 
+// Definition of an AI model file used in the workflow.
+const ModelFile = z.object({
```

Comment:
`ModelFile` => `zModelFile` for naming consistency.

Reply:
Done